packages/@aws-cdk/aws-cloudfront/lib/private/cache-behavior.ts

```diff
@@ -57,6 +57,9 @@ export class CacheBehavior {
           includeBody: edgeLambda.includeBody,
         }))
         : undefined,
+      trustedKeyGroups: this.props.trustedKeyGroups
+        ? this.props.trustedKeyGroups.map(keyGroup => keyGroup.keyGroupId)
+        : undefined,
```

Comment:
🙃  I see you did this in the other instance, but not here.

```suggestion
      trustedKeyGroups: this.props.trustedKeyGroups?.map(keyGroup => keyGroup.keyGroupId),
```

Reply:
Great catch... fixed.

Reply:
@njlynch I went ahead and fixed this one too...

before:
```ts
lambdaFunctionAssociations: this.props.edgeLambdas?
        ? this.props.edgeLambdas.map(edgeLambda => ({
          lambdaFunctionArn: edgeLambda.functionVersion.edgeArn,
          eventType: edgeLambda.eventType.toString(),
          includeBody: edgeLambda.includeBody,
        }))
        : undefined,
```

after:
```ts
lambdaFunctionAssociations: this.props.edgeLambdas?.map(edgeLambda => ({
        lambdaFunctionArn: edgeLambda.functionVersion.edgeArn,
        eventType: edgeLambda.eventType.toString(),
        includeBody: edgeLambda.includeBody,
      })),
```